F21/gademanasa/main.py

```diff
@@ -0,0 +1,37 @@
+class LowerCase:
+    def __init__(self, word, bcpCode):
+        self.word = word
+        self.l = bcpCode
+
+    def wordLower(self):
+        self.l=self.l[:2]
+        if self.l == 'tr' or self.l == 'az':

```

Comment:
what do you say about comparing the strings using 'as in' keyword which is easy to read, for eg: self.l as in tr